app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,9 @@
   </head>
 
   <body>
+  <p class="flash">
```

Comment:
we might want to revisit flash messages later on if we ever have multiple flash messages to display. as is, we'll have a big paragraph element so they aren't a big run-on string